Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -398,36 +498,28 @@ public virtual void OnDrawContent (Rectangle contentArea)
 
             foreach (View view in subviewsNeedingDraw)
             {
-                //view.Frame.IntersectsWith (bounds)) {
-                // && (view.Frame.IntersectsWith (bounds) || bounds.X < 0 || bounds.Y < 0)) {
                 if (view.LayoutNeeded)
                 {
                     view.LayoutSubviews ();
                 }
 
                 // Draw the subview
-                // Use the view's bounds (view-relative; Location will always be (0,0)
-                //if (view.Visible && view.Frame.Width > 0 && view.Frame.Height > 0) {
-                view.Draw ();
+                if (view.Title.Contains ("This label"))
+                { }
```

Comment:
This seems like a scam to me ;-)

Reply:
> This seems like a scam to me ;-)

Ooops. I do that fequently to set breakpoints. I find VS2022's conditional breakpoints hard to use.